src/shell/scripts/omp.ps1

```diff
@@ -69,7 +69,6 @@ New-Module -Name "oh-my-posh-core" -ScriptBlock {
             $command = $null
             $cursor = $null
             [Microsoft.PowerShell.PSConsoleReadLine]::GetBufferState([ref]$command, [ref]$cursor)
-            $command = ($command -split " ")[0]
```

Comment:
this was here because using tooltips in a continuation prompt failed. Can you validate that use-case?

Reply:
Yes, it's broken. Even without a continuation prompt context, it will fail when I have a double quote typed and then a space. 😲 But a single quote doesn't lead to an error. This is due to:

https://github.com/JanDeDobbeleer/oh-my-posh/blob/07e0d0e0c60c528e034de815e90222dfff32544f/src/shell/scripts/omp.ps1#L73

where it messes up the double quote nesting for arguments. We can fix this, just let me try. 😆

Reply:
@JanDeDobbeleer I have fixed this and re-validated it.